python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -295,6 +298,7 @@ class RandomForestClassifier(Base):
         cdef RandomForestMetaData[double, int] *rf_forest64 = \
             new RandomForestMetaData[double, int]()
         self.rf_forest64 = <size_t> rf_forest64
+        self.multi_class = 0
```

Comment:
Boolean would be better I believe. Also does this need to be a user accessible attribute  or should it be _multi_class? I believe it should be the latter, no?